src/plugins/es_ui_shared/docs/docs/form_lib/examples/dynamic_fields.md

```diff
@@ -0,0 +1,278 @@
+---
+id: dynamic_fields
+title: Dynamic fields
+sidebar_label: Dynamic fields
+---
+
+## Basic
+
+Dynamic fields are fields that the user can add or remove in your form. Those fields will end up in an array of _values_ or an array of _objects_.
```

Comment:
Would it be helpful to point back to the `UseArray` docs somewhere in this example? (https://github.com/elastic/kibana/blob/41ec445c92f2728f797fab757e7259f068d2297e/src/plugins/es_ui_shared/docs/docs/form_lib/core/use_array.md)

Reply:
Good point, I will add a link to the UseArray docs 👍